platform/mbed_rtc_time.h

```diff
@@ -34,6 +34,8 @@ struct timeval {
     time_t tv_sec;
     int32_t tv_usec;
 };
+#else
+#include <sys/time.h>
```

Comment:
Why is this needed?

Reply:
Fixed in fc422106e140aaddc7ef5e7869d45a966e3c78cc